### DISCUSSION TITLE
Breakpoint not working in dependency jar

### DISCUSSION THREAD

**Discussion by lyfne123** [scalameta/metals#3813] [General]
Hi, I'm a new learner of Metals, I have a question that my breakpoints in dependency jar is not working.

below is my example code:
```
object TestApp extends App {
  val a = ArrayBuffer(3, 2, 6)
  a.insert(1, 32)
  a ++= ArrayBuffer(5)
}
```

I can add breakpoint in TestApp, but if I add inside ++= function, which is in`org/scala-lang/scala-library/2.12.10/scala-library-2.12.10-sources.jar!/scala/collection/mutable/ArrayBuffer.scala`, line 98, the debugger can't hit that breakpoint.

I can see it at Run & Debug panel.
<img width="473" alt="截屏2022-04-11 下午5 25 39" src="https://user-images.githubusercontent.com/70868521/162709185-5e54581c-d86e-4ac1-bac9-05825bb3d2fb.png">

Could please anyone can reply my quesion? Thanks very much!

**Comment by lyfne123** [comment]
I'm running VS Code on macOS, with Metals 0.11.2 and Bloop v1.4.13, Scala version 2.12.10, sbt 1.3.13

**Comment by kpodsiad** [comment]
@lyfne123 I can confirm that breakpoints in the dependency jars are not working, but I haven't found a reason yet.

**Reply by lyfne123** [reply]
Sad to hear that, but thanks for your reply. I hope to see the bug fix in the near future!

**Reply by lyfne123** [reply]
And do I need to report an issue for this problem?

**Reply by tgodzik** [reply]
Created it, there is a handy feature for this. Thanks again for reporting!
